cmd/provisioning-client/bluetooth.go

```diff
@@ -30,6 +30,9 @@ const (
 	appAddressKey            = "app_address"
 	availableWiFiNetworksKey = "networks"
 	statusKey                = "status"
+	manufacturerKey          = "manufacturer"
+	modelKey                 = "model"
+	fragmentKey              = "fragment_id"
```

Comment:
Making a change on my end to read the fragment id, should i read the manufacturer and model for any reason? The only thing I could think of was for displaying it in the UI in some way (from my perspective)

Reply:
Think I got an answer here! https://github.com/viamrobotics/canyon-runner-mobile/pull/190#discussion_r2043267806

Reply:
These are purely informative for generic whitelabeling purposes. Safe to ignore, but I wanted to match parity with with web/grpc provisioning APIs.